src/NUnitEngine/nunit.engine.tests/Integration/RunnerInDirectoryWithoutFramework.cs

```diff
@@ -16,7 +16,7 @@ internal sealed class RunnerInDirectoryWithoutFramework : IDisposable
 
         public RunnerInDirectoryWithoutFramework()
         {
-            directory = new DirectoryWithNeededAssemblies("nunit3-console", "nunit-agent", "nunit-agent-x86");
+            directory = new DirectoryWithNeededAssemblies("nunit3-console", "nunit.engine", "nunit-agent", "nunit-agent-x86");
```

Comment:
Heh - took me a while to figure out why this change was needed. Guess that's confirmation that the structure is right! 😄

Reply:
Yeah, @jnm2's code for copying the directories only goes one level deep in following references, because that's all that was needed at the time. Rather than change it, I opted to just add the engine explicitly, thereby picking up core.